src/Blocks/components/form/assets/step.js

```diff
@@ -327,6 +327,16 @@ export class Steps {
 		);
 	}
 
+	toggleDebugPreview() {
+		const debug = document.querySelectorAll(this.state.getStateSelector('stepDebugPreview', true));
+
+		if (debug.length) {
+			[...debug].forEach((item) => {
+				item.addEventListener('click', this.onToggleDebugPreview);
+			});
+		}
```

Comment:
```suggestion
		const debug = document.querySelectorAll(this.state.getStateSelector('stepDebugPreview', true));

		debug.forEach((item) => item?.addEventListener('click', this.onToggleDebugPreview));
```
Since you're using `querySelectorAll`, you can be sure that you'll always get a `NodeList`, which is iterable just with `forEach` directly 😉 

![image](https://github.com/infinum/eightshift-forms/assets/77000136/e9b2fe0b-3f6d-486b-a2bc-a6a8ac43756e)